recipes/blossalg/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "blossalg" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65c2dd9b084659a20f338552eeea7dc0226a58b6bfc4e5a3e91475d6ad7ec77d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - future
```

Need a Python version for noarch:
```suggestion
    - python >=3.6
    - pip
  run:
    - python >=3.6
    - future
```
(assuming that's the right version range)